server/src/main/java/org/spine3/server/reflect/FailureSubscriberMethod.java

```diff
@@ -145,8 +145,8 @@ static FailureSubscriberMethod from(Method method) {
         return new FailureSubscriberMethod(method);
     }
 
-    private static IllegalStateException missingFailureHandler(Class<?> cls,
-                                                               Class<? extends Message> failureClass) {
+    private static IllegalStateException
+    missingFailureHandler(Class<?> cls, Class<? extends Message> failureClass) {
```

Comment:
I thinks, it's better to transfer a method parameters and keep a name at a previous position.